src/permission/config-schema.js

```diff
@@ -20,5 +20,10 @@ module.exports = {
 	rpc: {
 		provide: true,
 		request: true
+	},
+	presence: {
+		query: true,
+		logins: true,
+		logouts: true
```

Comment:
Or should we just have

presence: {
` ` query: true,
` ` sessions: true, //where sessions is both logins and logouts
}